src/main/java/org/fundacionjala/at15/katas/calculator/Calculator.java

```diff
@@ -2,7 +2,16 @@
 
 public class Calculator {
 
-    public int doOperation(Operation operation, int leftOperand, int rightOperand) {
+    public int doSimpleOperation(Operation operation, int leftOperand, int rightOperand) {
         return operation.run(leftOperand, rightOperand);
     }
+
+    public int doMoreOperation(Operation operation1, int leftOperand1, int rightOperand1, Operation operation2,
```

Comment:
This is a good approach, but can be improved, how could you make it work for multiple operations? Currently, this one only supports 2 operations

Reply:
I was trying to add more operations as parameters but the checkstyle gave me an error because there were many parameters. My next task will be add an array with many operations.